content/en/docs/concepts/cluster-administration/system-traces.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Traces For Kubernetes System Components
+reviewers:
+- logicalhan
+- lilic
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+System component traces record the latency of and relationships between operations in the cluster.
+
+Kubernetes components emit traces using the [OpenTelemetry Protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#opentelemetry-protocol-specification) with the gRPC exporter and can be collected and routed to tracing backends using the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector#-opentelemetry-collector).
+
+<!-- body -->
+
+## Stability
+
+Tracing instrumentation is still under active development, and may change in a variety of ways. This includes span names, attached attributes, instrumented endpoints, etc. There are currently no guarantees of backwards-compatibility for tracing instrumentation.
```

Comment:
I'd move this to end end of the page.
Also, maybe change:
> There are currently no guarantees of backwards-compatibility for tracing instrumentation.

to
> Until this feature graduates to stable, there are  no guarantees of backwards compatibility
> for tracing instrumentation.

Reply:
Done